### PR TITLE
feat: upgrade palette, use Spacer with x/y

### PR DESCRIPTION
**Pull request by pvinis**
The type of this PR is: **TYPE**

This PR solves [PROJECT-XX]

### Description

[GRO-434]: https://artsyproduct.atlassian.net/browse/GRO-434?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by pvinis** [comment]
I have left the ones I am not sure about unchanged, so type-check fails on them.

File: package.json [hunk 1]
```diff
@@ -80,8 +80,8 @@
     "@artsy/fresnel": "3.5.0",
     "@artsy/icons": "3.1.3",
     "@artsy/multienv": "^1.2.0",
-    "@artsy/palette": "26.1.2",
-    "@artsy/palette-charts": "25.1.2",
+    "@artsy/palette": "27.0.0-canary.1232.25371.0",
```
Comment (pvinis) [inline package.json:83]: this needs changing before merge

File: package.json [hunk 2]
```diff
@@ -80,8 +80,8 @@
...
+    "@artsy/palette-charts": "26.0.0-canary.1232.25371.0",
```
Comment (pvinis) [inline package.json:84]: this needs changing before merge

File: src/Apps/About/AboutApp.tsx [hunk 3]
```diff
@@ -27,42 +29,30 @@ export const AboutApp: React.FC = () => {
         imageURL="https://files.artsy.net/images/00_CVP_About_Hero_og.png"
         pathname="/about"
       />
-
-      <FullBleedHeader src="https://files.artsy.net/images/00_CVP_About_Hero%20(1).png">
+      <FullBleedHeader
+        src="https://files.artsy.net/images/00_CVP_About_Hero%20(1).png"
+        caption="Detail of Cassi Namoda, A Strange Song, 2022. Detail of Alex Katz,
+            Day Lily 1, 1969."
+      >
```
Comment (dzucconi) [inline src/Apps/About/AboutApp.tsx:36]: I fixed a few random things in the about app; couldn't help myself.

File: src/Apps/ArtQuiz/ArtQuizWelcome.tsx [hunk 4]
```diff
@@ -26,55 +25,54 @@ export const ArtQuizWelcome: FC<ArtQuizWelcomeProps> = ({ onStartQuiz }) => {
...
+              as={RouterLink}
+              variant="tertiary"
+              width="100%"
+              to="/"
+            >
+              {t("artQuizPage.welcomeScreen.skipButton")}
+            </Button>
+          </Flex>
+        }
+      />
```
Comment (dzucconi) [inline src/Apps/ArtQuiz/ArtQuizWelcome.tsx:75]: OK I should have left this one alone to minimize the conflict; lemme roll that off

File: src/Apps/Artwork/Components/ArtworkImageBrowser/ArtworkActions.tsx [hunk 5]
```diff
@@ -164,7 +164,7 @@ export const ArtworkActions: React.FC<ArtworkActionsProps> = ({
       )}
 
       <Container>
-        <Join separator={<Spacer mx={0} />}>
+        <Join separator={<Spacer x={0} />}>
```
Comment (dzucconi) [inline src/Apps/Artwork/Components/ArtworkImageBrowser/ArtworkActions.tsx:167]: Go figure.

**Comment by damassi** [comment]
Nice work y'all

**Pull request merged**